hazelcast/src/test/java/com/hazelcast/internal/util/phonehome/PhoneHomeIntegrationTest.java

```diff
@@ -163,5 +167,75 @@ public void testCacheMetrics() {
                 .withQueryParam("cawact", equalTo("1")));
     }
 
+    @Test
+    public void testMapLatencies() {
+        DelayMapStore mapStore = new DelayMapStore();
+        MapStoreConfig mapStoreConfig = new MapStoreConfig();
+        mapStoreConfig.setEnabled(true);
+        mapStoreConfig.setClassName(getClass().getName());
+        mapStoreConfig.setImplementation(mapStore);
```

Comment:
nit: you can avoid the mapStore local variable (just `setImplementation(new DelayMapStore())` works)